<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(51, 51, 51);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>190</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">  font: 12pt &quot;Segoe UI&quot;;
  color: rgb(193, 193, 193)</string>
    </property>
    <property name="text">
     <string>Usuario</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditUsuario">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>220</y>
      <width>301</width>
      <height>28</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{
font: 12pt &quot;Segoe UI&quot;;
color: rgb(0, 0, 0);
border: 1px solid #ccc;
border-radius: 10px;
background-color: rgb(194, 194, 194);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditPassword">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>300</y>
      <width>301</width>
      <height>28</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{
font: 12pt &quot;Segoe UI&quot;;
color: rgb(0, 0, 0);
border: 1px solid #ccc;
border-radius: 10px;
background-color: rgb(194, 194, 194);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>270</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">  font: 12pt &quot;Segoe UI&quot;;
  color: rgb(193, 193, 193)</string>
    </property>
    <property name="text">
     <string>Contrase√±a</string>
    </property>
   </widget>
   <widget class="QPushButton" name="registrar">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>370</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
  font: 12pt &quot;Segoe UI&quot;;
  border: none;
  background-color: rgb(68, 68, 68);
  color: white;
  border-radius: 10px;
}

QPushButton:hover {
  background-color: rgb(81, 81, 81);
  border-color: rgb(81, 81, 81);
}

QPushButton:pressed {
  background-color: rgb(81, 81, 81);
  border-color: rgb(81, 81, 81);
}
</string>
    </property>
    <property name="text">
     <string>Registrar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ingresar">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>370</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
  font: 12pt &quot;Segoe UI&quot;;
  border: none;
  background-color: rgb(68, 68, 68);
  color: white;
  border-radius: 10px;
}

QPushButton:hover {
  background-color: rgb(81, 81, 81);
  border-color: rgb(81, 81, 81);
}

QPushButton:pressed {
  background-color: rgb(81, 81, 81);
  border-color: rgb(81, 81, 81);
}
</string>
    </property>
    <property name="text">
     <string>Ingresar</string>
    </property>
   </widget>
   <widget class="QLabel" name="warningMain">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>440</y>
      <width>371</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">  font: 12pt &quot;Segoe UI&quot;;
  color:rgb(225, 116, 39);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_33">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>50</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">  font: 16pt &quot;Segoe UI&quot;;
  color: rgb(193, 193, 193);
font-weight: bold;</string>
    </property>
    <property name="text">
     <string>GrowthSphere</string>
    </property>
   </widget>
   <widget class="QLabel" name="logo_main">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>30</y>
      <width>91</width>
      <height>81</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>e.jpg</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1280</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
